### PR TITLE
feat(ui): Centralize fonts and clean footer themes

### PR DESCRIPTION
**Pull request by rrchai**
This PR depends on #626

### CODE REVIEW

**Review by vpchung — APPROVED** [review]

File: apps/challenge-registry/src/_app-theme.scss [hunk 1]
```diff
@@ -1,24 +1,24 @@
 @use '@angular/material' as mat;
-@use 'palette' as pal;
+@use 'libs/challenge-registry/themes/src/lib/fonts' as fonts;
+@use 'libs/challenge-registry/themes/src/lib/palettes' as pals;
```
Comment (tschaffter) [inline apps/challenge-registry/src/_app-theme.scss:3]:
```suggestion
@use 'libs/challenge-registry/themes/src/lib/palettes' as palettes;
```
Using `palettes` would be more descriptive, unless `pals` is commonly used somewhere else (e.g. Angular Material docs)

File: libs/challenge-registry/home/src/_lib-theme.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+@use './lib/featured-challenge-list/featured-challenge-list-theme' as featured;
```
Comment (tschaffter) [inline libs/challenge-registry/home/src/_lib-theme.scss:1]: Consider using a more descriptive name, e.g. `featuredChallengeList` or `featured-challenge-list`.

**Review by tschaffter — COMMENTED** [review]
This looks great! I left two comments about using more descriptive/standardized variable names, which can maybe be extended to other variables.

File: libs/challenge-registry/home/src/_lib-theme.scss (hunk 2 above)
Comment (rrchai) [reply libs/challenge-registry/home/src/_lib-theme.scss]: has changed to `featured-challenge-list`

**Comment by rrchai** [comment]
TODO:
- [ ] (@vpchung suggests ) Try to use `typography-config` to see if the fonts will automatically configured for all pages. If so, we don't need to re-specify the typography styles in each component.

**Pull request merged**